libs/sections/src/header/vl-header.section.ts

```diff
@@ -66,6 +74,11 @@ export class VlHeader extends BaseLitElement {
         super.disconnectedCallback();
 
         this.headerContainer?.remove();
+
+        if (this.allowSkeleton) {
+            this.headerContainerSkeleton?.remove();
```

Comment:
op zich kan het geen kwaad om `this.headerContainerSkeleton?.remove();` in elk geval uit te voeren, gezien er een null check gebeurt op `this.headerContainerSkeleton`